tox.ini

```diff
@@ -1,9 +1,9 @@
 # content of: tox.ini , put in same dir as setup.py
 [tox]
-envlist = py26,py27,py34
+envlist = py27,py34,py35
```

Comment:
The changes made to use TLS 1.2 are not compatible with python 2.6. Removed python 2.6 from list of test environments.